.github/workflows/pr_and_main.yml

```diff
@@ -0,0 +1,76 @@
+name: PR and main CI
+
+on: 
+  schedule:
+    - cron: '0 1 * * *'
```

Comment:
Is this a setup for a nightly build?

Reply:
Yes.

Reply:
I now realize that I mixed this up... we need the nightly build for the package build, I'll fix tomorrow.